README.md

```diff
@@ -217,9 +217,7 @@ export default Ember.Controller.extend({
           [
             {
               model: 'password',
-              properties: {
-                type: 'password' // Render input as a password input instead of default text input
-              }
+              renderer: 'password'
```

Comment:
Thanks!

I tried it but getting this error:
#/cellDefinitions/main/children/2/children/0/renderer Data does not match any schemas from 'oneOf'

I'm also guessing the the custom 'name-renderer' will have a similar change?